drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInActivityEvent.kt

```diff
@@ -9,11 +9,23 @@
 package com.adyen.checkout.dropin.ui.viewmodel
 
 import com.adyen.checkout.components.PaymentComponentState
+import com.adyen.checkout.components.model.paymentmethods.PaymentMethod
 import com.adyen.checkout.components.model.payments.request.OrderRequest
+import com.adyen.checkout.components.model.payments.response.Action
+import com.adyen.checkout.dropin.ui.giftcard.GiftCardPaymentConfirmationData
 
 sealed class DropInActivityEvent {
     data class MakePartialPayment(val paymentComponentState: PaymentComponentState<*>) : DropInActivityEvent()
     object ShowPaymentMethods : DropInActivityEvent()
     class CancelOrder(val order: OrderRequest, val isDropInCancelledByUser: Boolean) : DropInActivityEvent()
     object CancelDropIn : DropInActivityEvent()
+    class LoadFragment(val fragment: DropInFragmentToLoad) : DropInActivityEvent()
```

Comment:
I would do something like this:
```
...
    class NavigateTo(val destination: DropInDestination) : DropInActivityEvent()
}

internal sealed class DropInDestination {
     object PreselectedStored : DropInDestination()
     object PaymentMethods : DropInDestination()
     class PaymentComponent(val paymentMethod: PaymentMethod) : DropInDestination()
     class ActionComponent(val action: Action) : DropInDestination()
     class GiftCardPaymentConfirmation(val data: GiftCardPaymentConfirmationData) : DropInDestination()
 }
```
That way it's also independent of wether a fragment or activity uses it and it tells a little better what is going to happen imo